vagrant/centos8.ks

```diff
@@ -0,0 +1,147 @@
+#url --mirrorlist=http://mirrorlist.centos.org/?release=8&arch=x86_64&repo=BaseOS&infra=stock
+#repo --name=AppStream --mirrorlist=http://mirrorlist.centos.org/?release=8&arch=x86_64&repo=AppStream&infra=stock
+install
+text
+keyboard us
+lang en_US.UTF-8
+skipx
+network --bootproto dhcp
+rootpw --plaintext vagrant
+firewall --disabled
+authconfig --enableshadow --enablemd5
+selinux --enforcing
+timezone --utc UTC
+services --enabled=vmtoolsd,NetworkManager,sshd
```

Comment:
I think 'services --enabled=vmtoolsd' is sufficient, the others will be enabled anyway